emperor/support_files/js/scale-view-controller.js

```diff
@@ -0,0 +1,278 @@
+define([
+    'jquery',
+    'underscore',
+    'util',
+    'viewcontroller',
+    'scale-editor'
+], function($, _, util, ViewControllers, ScaleEditor) {
+
+  // we only use the base attribute class, no need to get the base class
+  var EmperorAttributeABC = ViewControllers.EmperorAttributeABC;
+
+  /**
+   * @class ScaleViewController
+   *
+   * Alters the scale of points displayed on the screen.
+   *
+   * @param {Node} container Container node to create the controller in.
+   * @param {Object} decompViewDict This object is keyed by unique
+   * identifiers and the values are DecompositionView objects referring to a
+   * set of objects presented on screen. This dictionary will usually be shared
+   * by all the tabs in the application. This argument is passed by reference.
+   *
+   * @return {ScaleViewController}
+   * @constructs ScaleViewController
+   * @extends EmperorAttributeABC
+   *
+   **/
+  function ScaleViewController(container, decompViewDict) {
+    var helpmenu = 'Change the scale of the attributes on the plot, allowing ' +
+                   'highlighting of points using size.';
+    var title = 'Scale';
+    // Create checkbox for scaling by values
+    /**
+     * jQuery node for checkbox controlling whether to scale by values or not
+     * @type {Node}
+     */
+    this.$scaledValue = $('<input type="checkbox">');
+    /**
+     * jQuery node for label of $scaledValues
+     * @type {Node}
+     */
+    this.$scaledLabel = $('<label for="scaledValue">Scale by values</label>');
```

Comment:
Originally it allowed for clicking the label as well to check the box, but since we're not using ids for things I can remove it.